.github/workflows/cd.yml

```diff
@@ -6,8 +6,8 @@ on:
       - v*
 
 env:
-  GO_VERSION: '1.17'
```

Comment:
If you change

```
      - name: Setup Go
        uses: actions/setup-go@v2
```

to 

```
      - name: Set up Go
        uses: actions/setup-go@v3
        with:
          go-version-file: 'go.mod'
```

then you can get rid of this `GO_VERSION` env variable, and future bumps to go versions will be easier, by just changing directly in `go.mod`

Reply:
good point